app/views/my/_webconference_room.html.haml

```diff
@@ -34,11 +34,9 @@
           - if can?(:end, @room)
             = link_to t('.end'), end_bigbluebutton_room_path(@room, :redir_url => my_home_path), options_for_tooltip(t('.end_tooltip'), :data => { :confirm => t('are_you_sure') }, :class => 'btn btn-small btn-danger')
         - else
-          -# only show the options dialog if the user can record and the record flag is not automatically set
-          - if can?(:record_meeting, @room) && !current_site.webconf_auto_record
-            = link_to t('.start'), join_options_bigbluebutton_room_path(@room), :class => "btn btn-small open-modal btn-primary", :'data-modal-width' => 'small'
-            = link_to join_bigbluebutton_room_path(@room), options_for_tooltip(t(".fast_start"), :class => "btn btn-small btn-primary webconf-join-link") do
-              = icon_webconf_start
+          -# only show the options dialog if the user can record
+          - if can?(:record_meeting, @room)
```

Comment:
This `if` is not necessary, the code in the `if` and in the `else` are the same.